### ISSUE TITLE
user module should be able to configure RBAC attributes on Solarish systems

### ISSUE THREAD

**Issue by xen0l** [ansible/ansible-modules-core#3936]
##### ISSUE TYPE
- Feature Idea
##### COMPONENT NAME

user
##### ANSIBLE VERSION

```
2.1.0.0
```
##### CONFIGURATION

Default
##### OS / ENVIRONMENT

OpenIndiana/SmartOS/Oracle Solaris
##### SUMMARY

illumos/Solaris RBAC provides authorizations/profiles/roles attributes. These should be configurable via Ansible
##### STEPS TO REPRODUCE

```
- name: Configure RBAC profiles
  user: name=xenol
           profile='Primary Administrator'

- name: Configure RBAC authorization
  user: name=xenol
           authorization='zfssnap'

- name: Configure RBAC role
  user: name=xenol
           role='root'
```
##### EXPECTED RESULTS

RBAC authorization/profile/role will be assigned to the user account.

**Comment by alikins** [comment]
ping @grossws

**Comment by grossws** [comment]
I have no Solaris/Illumos systems, so it out of scope of my interests currently.

Feel free to make a pull request if you want to contribute a patch.

**Comment by grossws** [comment]
@ansibot, needs_contributor

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by xen0l** [comment]
This issue was moved to ansible/ansible#19179
